src/counter_analysis_toolkit/main.c

```diff
@@ -320,10 +320,10 @@ static hw_desc_t *obtain_hardware_description(char *conf_file_name){
     hw_desc->split[1] = 1;
     hw_desc->split[2] = 1;
     hw_desc->split[3] = 1;
-    hw_desc->pts_per_reg[0] = 5;
-    hw_desc->pts_per_reg[1] = 5;
-    hw_desc->pts_per_reg[2] = 5;
-    hw_desc->pts_per_reg[3] = 5;
+    hw_desc->pts_per_reg[0] = 3;
+    hw_desc->pts_per_reg[1] = 3;
+    hw_desc->pts_per_reg[2] = 3;
+    hw_desc->pts_per_reg[3] = 3;
```

Comment:
In the commit message do you mean "[...] increase the benchmark execution speed" rather than "[...] decrease the benchmark execution speed" ?

Reply:
Yes, I meant to write "decrease the benchmark execution time." I will update this commit message.

Reply:
This has been updated.